libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/MapboxTripStarter.kt

```diff
@@ -41,7 +46,8 @@ class MapboxTripStarter internal constructor() : MapboxNavigationObserver {
         ReplayRouteSessionOptions.Builder().build()
     )
     private val isLocationPermissionGranted = MutableStateFlow(false)
-    private var replayRouteTripSession: ReplayRouteSession? = null
+    private var replayRouteSession: ReplayRouteSession? = null
```

Comment:
Is it possible to make `replayRouteSession` final and not null like `replayHistorySession`?

Reply:
Yeah but I think that should be done separately. The route session relies on the constructor right now to reset the clock https://github.com/mapbox/mapbox-navigation-android/blob/09b52480c921d4acb03e206da778d4a4735a6477/libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route/ReplayRouteDriver.kt#L17